metrics/metrics-core/src/test/java/org/apache/servicecomb/metrics/core/TestAnMonitorManager.java

```diff
@@ -41,7 +41,7 @@
 
   private static MetricsLoader currentWindowMetricsLoader;
 
-  private static MetricsLoader nextWindowMetricsLoader;
+  //  private static MetricsLoader nextWindowMetricsLoader;
```

Comment:
Why did you comment out these code? 
If we don't need them, we can just remove them.

Reply:
cause random UT failed.
this class, and related classes will be deleted in the next PR